com.nitrokey.nitrokey-app.yml

```diff
@@ -0,0 +1,60 @@
+app-id: com.nitrokey.nitrokey-app
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: nitrokey-app
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --filesystem=host
+  - --device=all
+modules:
+  - name: libusb
+    buildsystem: autotools
```

Comment:
`autotools` is the default build system and doesn't need to be set explicitly.

Reply:
done